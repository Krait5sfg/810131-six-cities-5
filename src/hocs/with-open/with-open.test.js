import React from 'react';
import renderer from 'react-test-renderer';
import withOpen from './with-open';

const MockComponent = () => {
  return <div />;
};

const MockComponentWrapped = withOpen(MockComponent);

it(`withOpen is render correctly`, () => {
  const tree = renderer.create((
    <MockComponentWrapped
      isOpen={true}
      onSortingClick={() => {}}
    />
  )).toJSON();

  expect(tree).toMatchSnapshot();
});
