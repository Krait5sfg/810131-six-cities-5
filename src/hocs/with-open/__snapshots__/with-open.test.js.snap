// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withOpen is render correctly 1`] = `<div />`;
